clustering/jgroups/extension/src/test/resources/org/jboss/as/clustering/jgroups/subsystem/subsystem-jgroups-transform-reject.xml

```diff
@@ -75,12 +76,7 @@
             </relay>
         </stack>
         <stack name="minimal">
-            <transport type="UDP">
-                <default-thread-pool min-threads="11"/>
-                <internal-thread-pool queue-length="22"/>
-                <oob-thread-pool keepalive-time="34"/>
-                <timer-thread-pool max-threads="43"/>
```

Comment:
Well, the rejections are still being tested in the "maximal" stack (i.e. https://github.com/wildfly/wildfly/pull/8978/files/bc5870ef9e54db25613e3dbff253d682af777329#diff-4e1952a60eaff7ba9b44979ec09e94c4R43), so some rejection test are still there. Hm, I don't remember why I added these here anyway.